app/components/WebsocketProvider.tsx

```diff
@@ -352,7 +352,12 @@ class WebsocketProvider extends React.Component<Props> {
         comment?.updateReaction({
           type: "add",
           emoji: event.emoji,
-          userId: event.userId,
+          userId: event.user.id,
+        });
+        comment?.updateReactedUser({
```

Comment:
Why have this separate? Any reason we'd call `updateReaction` but not `updateReactedUser`?

Reply:
Not really.. can be consolidated into a single function 👍